packages/ui/ButtonGroup/index.tsx

```diff
@@ -137,3 +122,24 @@ const ButtonGroupStyled = styled.div<GroupProps>`
           `}
   }
 `;
+
+export const ButtonGroup = ({ options, secondary }: GroupProps) => {
+  return (
+    <ButtonGroupStyled options={options} secondary={secondary}>
+      {options.map(({ component, onSelect, defaultChecked, id }, index) => {
+        return (
+          <Fragment key={id}>
+            <input
+              type={"radio"}
+              id={`button-${index}`}
+              onClick={onSelect}
+              name={"button-group"}
```

Comment:
you don't need brackets when passing strings
```suggestion
              type="radio"
              id={`button-${index}`}
              onClick={onSelect}
              name="button-group"
```

Reply:
it's not my code, I just moved the styled components to the top, but I can improve it of course :)

Reply:
done